spanner/value.go

```diff
@@ -877,6 +877,98 @@ func (n *PGNumeric) UnmarshalJSON(payload []byte) error {
 	return nil
 }
 
+// NullProto represents a Cloud Spanner PROTO that may be NULL.
+type NullProto struct {
```

Comment:
We have been using ProtoMessage for proto in previous PR. Should we be consistent here?

Reply:
@Shua1 
I have updated `NullProto` to be `NullProtoMessage` and `NullEnum` to be `NullProtoEnum`